pom.xml

```diff
@@ -4,7 +4,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>com.channelape</groupId>
 	<artifactId>shopify-sdk</artifactId>
-	<version>1.3.0-SNAPSHOT</version>
+	<version>1.4.0.RETRY_CONFIGURATION-SNAPSHOT</version>
```

Comment:
@rjdavis3 this wasn't a breaking change since the build step is just absorbed into the optionals step.